File: libraries/c#wasdilib/WasdiLib/Models/WorkspaceEditorViewModel.cs
namespace WasdiLib.Models
{
    internal class WorkspaceEditorViewModel
    {

        public string WorkspaceId { get; set; }
        public string Name { get; set; }
        public string UserId { get; set; }
        public string ApiUrl { get; set; }
        public Int64 CreationDate { get; set; }
        public Int64 LastEditDate { get; set; }
        public List<string> SharedUsers { get; set; }
        public string NodeCode { get; set; }
        public long ProcessCount { get; set; }
        public string CloudProvider { get; set; }
        public string SlaLink { get; set; }

    }
}

File: libraries/jswasdilib/src/viewModels/WorkspaceEditorViewModel.ts


export class WorkspaceEditorViewModel {
    private workspaceId!: string;
    private name!: string;
    private userId!: string;
    private apiUrl!: string;
    private creationDate!: Date;
    private lastEditDate!: Date;
    private sharedUsers!: string[];
    private nodeCode!: string;
    private activeNode!: boolean;
    private processesCount!: number;
    private cloudProvider!: string;
    private slaLink!: string;
    private isPublic: boolean = false;
    private readOnly: boolean = false;

    constructor(){}
}